### ISSUE TITLE
Big hashrate drop in ver.2.5.2

### ISSUE THREAD

**Issue by GrannyCryptomaster** [xmrig/xmrig-nvidia#141]
I tryed today the new 2.5.2 version with ETN mining, on hashvault. I used the official config generator, I changed the threads to 32 from 26. So the settings looks like this 32x27 6x25.
I see a new parameter "variant -1", that I understand is for a new pow... In rest, nothing changed in config or in OC.
I have 6 **GTX 1060 3GB** hynix mems, OC: power 70, temp 83, core -100, mem +500; monitor in mobo port. Win 10 x64bit, last cuda 9, nvidia drivers, vc-redist and win updates.
I get around **330 h/s**, with a total of 2010 h/s. 
With the same settings, in 2.4.5 version, I got around **480 h/s**/card, 2880 h/s in total, 3 weeks ago.
Why such a big hashrate difference? Is from the new pow? Did ETN changed the pow? 
Or the miner performs badly?

**Comment by tudde** [comment]
Since Gtx 1060 has 18 SMPs I think you should use number of blocks that is a multiple of 18, so you might try 32x36, which would use more memory. 32x36 equals 1152 which is the number of cores on your card. What hashrate do you get with that setting?

**Comment by tudde** [comment]
Ok, I used information from Videocardz.net but they are mistaken claiming that gtx 1060 has 18 multiprocessors. Thanks for correction. Now I do remember that you have mentioned that in other thread earlier. :)

**Comment by GrannyCryptomaster** [comment]
I found this: "https://bitcointalk.org/index.php?topic=2126975.0".
But I don't understand that formula and how to use it... (it's a formula for threads and blocks, at xmrig nvidia section, in first post).
I know my compute power is 6.1 for this card, and has Pascal GPU, so the RAM limit is 16 GB on CUDA 9. But how to use this in that formula? From where he took 2 and 1900? Maybe you can make sense of it...

**Comment by GrannyCryptomaster** [comment]
...and regardless of threads and blocks, because I used the same parameters with both versions, why such drop in hash rates???

**Comment by xmrig** [comment]
Please make make screenshot or copy miner summary as text, for both versions with same config. Also with per card hashrate.

Old and new version should give same hashrate, because used miner code, no major changes happen.

Some differences:
 * Old version built with CUDA 9.0 new with 9.1.
* Autoconfig changed, but if you use same config it doesn't mater.

Thank you.

**Comment by GrannyCryptomaster** [comment]
I made tests with 2 nVidia drivers, because the last driver from nVidia was crashing ethminer on ETH mining. I have CUDA 9.1 installed, so I used as old driver, the 388.71 version, the first one with CUDA 9.1 support. 
Rig: 6x GTX 1060 3GB hynix, Win 10 x64 up-to-date, CUDA 9.1, vc-redist all versions including 2017, OC Afterburner power 70, temp 83, core -100, mem +500. I recommend using core 0 or an aproppiate value for cryptonights, cause it gives a few more hashes then -xxx, but I sticked with -100 cause I mine ethash coins at the moment. The best hashrates I got a month ago was 480-490 H/s on all cards with core OC.
Here are the results:
{
    "algo": "cryptonight",
    "background": false,
    "colors": true,
    "donate-level": 1,
    "log-file": null,
    "print-time": 300,
    "retries": 200,
    "retry-pause": 5,
    "syslog": false,    
    "threads": [
        {
            "index": 0,
            "threads": 32,
            "blocks": 27,
            "bfactor": 6,
            "bsleep": 25,
            "affine_to_cpu": false
        },
      ......(silimar settings for other 5 GPUs).....
    "pools": [
        {
            "url": "pool.electroneum.hashvault.pro:3333",
            "user": "xxxxxxxxxxxxxxxxxxxx",
            "pass": "xxxxxxxxxxxx",
            "keepalive": true,
            "nicehash": false,

        }
    ],
    "api": {
        "port": 0,
        "access-token": null,
        "worker-id": null
    }
}

![xmrig 245 - driver 388 71 1](https://user-images.githubusercontent.com/34797446/38170714-e50c97c2-3594-11e8-8881-bbe875167e5b.png)
![xmrig 252 - driver 388 71 2](https://user-images.githubusercontent.com/34797446/38170725-10cefb0c-3595-11e8-998d-09e15dcd0948.png)
![xmrig 252 - driver 391 35 3](https://user-images.githubusercontent.com/34797446/38170726-25287c5e-3595-11e8-84c0-4c7796254a8b.png)
![afterburbner](https://user-images.githubusercontent.com/34797446/38170730-3ad89e08-3595-11e8-9273-294fec9b4241.png)

**Comment by wizdude** [comment]
i've noticed a big drop in hash rate as well. i've tried using the "auto" config and it's not helped. dropped by around 50% on some machines.

i'm going to continue experimenting, but i wanted to know if the issue was related to xmrig or something to do with the new hashing algorithm?

**Comment by GrannyCryptomaster** [comment]
OK. I found the problem, and the hashrate is where is supouse to, back to 480/card. 
For GTX 1060 3GB hynix mem, stick with 32x27 8x100, core 0 or -x, mem +500, power 70, 
in WINDOWS 10 x64.
CUDA 9.1 + all patches, nVidia driver 388.71.
The hashrate drop was caused by bfactor and bsleep; I was using 6x25 - the performance settings from autoconfig. In Windows, even if you have the monitor pluged in mobo, not in gpu, you must use 8x100! Thats the "smooth desktop" settings.
